timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -2046,37 +2024,21 @@ PYBIND11_MODULE(custom_ops, m) {
         py::arg("group_idxs"),
         py::arg("radius"));
     m.def(
-        "translation_within_sphere_f32",
-        &py_translation_within_sphere<float>,
+        "translations_inside_and_outside_sphere_host_f32",
```

Comment:
nit: any reason to use different names in the Python API? (with and without `get_` prefix)?

Reply:
Only reason here is to be consistent with the other wrapped calls.

Reply:
Changed the C++ function name to be consistent with the wrapped code in https://github.com/proteneer/timemachine/pull/1234/commits/46a8cd4a58a31abc02cae152b3a20e57cf6dbf36